### PR TITLE
Removes Merc job

### PR DESCRIPTION
**Pull request by namjay1170**
Merc job entirely removed, not commented out.

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@
 BEFORE ADDING ADMINS TO FALLOUT 13 READ ADMIN_RANKS.TXT
 ###############################################################################################
 Xarvox = President
+Namjay1170 = President
```
Comment (Angryon) [inline config/admins.txt]: Uh-uh.

File: code/game/jobs/jobs.dm [hunk 2]
```diff
@@ -38,7 +38,6 @@ var/const/AI				=(1<<6)
 var/const/CYBORG			=(1<<7)
 
 var/const/WASTELAND			=(1<<5)
-var/const/MERCENARY			=(1<<0)
 var/const/CULTLEADER		=(1<<1)
```
Comment (Angryon) [inline code/game/jobs/jobs.dm]: can you cut out the messiah while you're at it i don't think anyone's ever actually used that

File: config/admins.txt (hunk 1 above)
Comment (namjay1170) [inline config/admins.txt]: Seems i autistic and didnt discard that.

File: code/game/jobs/jobs.dm (hunk 2 above)
Comment (AndrewL97) [inline code/game/jobs/jobs.dm]: +1

File: config/admins.txt (hunk 1 above)
Comment (AndrewL97) [inline config/admins.txt]: thats a big ol no no

**Pull request closed without merging**